hpc-job-submission.md

```diff
@@ -122,7 +122,84 @@ view all jobs for a particular partition with `squeue -p univ`.
 **3. Viewing Additional Job Information**
 ==================
 
-TBD: using `sacct`
+Accounting information for jobs that are invoked with SLURM are logged. The `sacct`	 command displays job accouting data in a variety of forms for your analysis. 
+
+<!-- Show basic sacct command run on account with jobs running -->
+
+## How To Select Jobs
+
+- To display information about a specific job or list of jobs use `-j` or `--jobs` Followed by a job number or comma seperated list of job numbers.
+	
+	```
+	[alice@login]$ sacct -jobs job1,job2,job3
+	```
+	{:.term}
+<!-- Sample output -->
+
+- To select information about jobs in a certian date range use `--start` and `--end` Without it, `sacct` will only return jobs from the current day.
+	
+	```
+	[alice@login]$ sacct --start=YYYY-MM-DD
+	```
+	{:.term}
+
+- To select information about jobs in a certian time range use `--starttime` and `--endtime` The default start time is 00:00:00 of the current day, unless used with `-j` then the default start time is Unix Epoch 0. The default end time is time of running the command Valid time formats are
+	```
+	HH:MM[:SS] [AM|PM]
+	MMDD[YY] or MM/DD[/YY] or MM.DD[.YY]
+	MM/DD[/YY]-HH:MM[:SS]
+	YYYY-MM-DD[THH:MM[:SS]] 
+	```
+
+	```
+	[alice@login]$ sacct --starttime 08/23 --endtime 08/24
+	```
+	{:.term}
+
+- To display another user's jobs use `--user`
+	
+	```
+	[alice@login]$ sacct --user BuckyBadger
+	```
+	{:.term}
+	<!-- Sample output -->
+
+- To only show statistics relevant to the job allocation itself, not taking steps into consideration use `-X`. This can be useful when trying to figure out which part of a job errored out.
+	
+	```
+	[alice@login]$ sacct -X
+	```
+	{:.term}
+	<!-- Sample Output -->
+
+## Displaying Specific Fields
+
+Sacct can display different fields about your jobs. You can use the `--helpformat` flag to get a full list.
+
+```	
+[alice@login]$ sacct --helpformat
+```
+{:.term}
+
+### Recommended Fields
+
+When looking for information about your jobs CHTC reccomends using these fields
+```
+elapsed
+end
+exitcode
+jobid
+ncpus
+nnodes
+nodelist
+ntasks
+partition
+start
+state
+submit
+user
+```
+
```

Comment:
Can you add something here like "a sample `sacct` command that prints out when a job started, ended, and how many nodes and cpus it requested would like like this:" and then have an example?